deluge/ui/gtkui/common.py

```diff
@@ -75,7 +75,7 @@ def build_menu_radio_list(value_list, callback, pref_value=None, suffix=None, sh
         item_text = str(value)
         if suffix:
             item_text += " " + suffix
-        menuitem = gtk.RadioMenuItem(group, item_text)
+        menuitem = Gtk.RadioMenuItem(item_text, group)
```

Comment:
without this  change and the change you commented on bellow the status bar makes menus that instead of values just say <RadioMenuItem object @ (GtkRadioMenuItem @)>